.github/workflows/build.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         python setup.py sdist bdist_wheel
     - name: Publish a Python distribution to PyPI
       if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
-      uses: pypa/gh-action-pypi-publish@master
+      uses: pypa/gh-action-pypi-publish@release/v1
       with:
         user: __token__
```

Comment:
Thanks for updating this. It seems like the `user` value is set to the default so perhaps we can leave it out? https://github.com/pypa/gh-action-pypi-publish#specifying-a-different-username